api/users/users.proto

```diff
@@ -23,4 +23,5 @@ message UserReferrer {
 	bytes ethereum_address = 1;
 	// referrer_valid will be false if the referring user has deleted their account or no longer has a confirmed eth address
 	bool referrer_valid = 2;
+	string id = 3;
```

Comment:
It’s probably unwise for this to not be optional, but it’s consistent with what’s been done for address.